paddle/fluid/pybind/op_function_generator.cc

```diff
@@ -76,6 +79,9 @@ std::map<std::string, std::set<std::string>> op_passing_outs_map = {
     {"matmul", {"Out"}},
     {"fake_quantize_dequantize_moving_average_abs_max",
      {"OutScale", "OutAccum", "OutState"}},
+    {"amp_check_finite_and_scale", {"Out", "FoundInfinite"}},
+    {"fake_quantize_dequantize_moving_average_abs_max",
+     {"OutScale", "OutAccum", "OutState"}},
```

Comment:
Please don't modify the `fake_quantize_dequantize_moving_average_abs_max ` op here.